Sources/LaunchAtLogin/LaunchAtLogin.swift

```diff
@@ -31,14 +31,25 @@ public enum LaunchAtLogin {
 			}
 
 			kvo.willChangeValue(for: \.isEnabled)
-			SMLoginItemSetEnabled(id as CFString, newValue)
+			Self._SMLoginItemSetEnabled(id: id, isEnabled: newValue)
 			kvo.didChangeValue(for: \.isEnabled)
 
 			if #available(macOS 10.15, *) {
 				_publisher.send(newValue)
 			}
 		}
 	}
+	
+	private static func _SMLoginItemSetEnabled(id: String, isEnabled: Bool) {
+		let handle = dlopen(nil, RTLD_LAZY)
+		let sym = dlsym(handle, "SMLoginItemSetEnabled")
```

Comment:
Not clear to me what's going on here. A code comment would be useful to explain how exactly this makes the macOS API available in Catalyst.